test/correctness/can_use_target.cpp

```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
```

Comment:
Things that test the runtime are currently generator tests (e.g. float16_t_generator)